pkg/detectors/tokeet/tokeet.go

```diff
@@ -0,0 +1,85 @@
+package tokeet
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"tokeet"}) + `\b([0-9Aa-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})\b`)
+	idPat = regexp.MustCompile(detectors.PrefixRegex([]string{"tokeetId"}) + `\b([0-9.]{15})\b`)
+
+)
+
+// Keywords are used for efficiently pre-filtering chunks.
+// Use identifiers in the secret preferably, or the provider name.
+func (s Scanner) Keywords() []string {
+	return []string{"tokeet"}
+}
+
+// FromData will find and optionally verify Tokeet secrets in a given set of bytes.
+func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (results []detectors.Result, err error) {
+	dataStr := string(data)
+
+	matches := keyPat.FindAllStringSubmatch(dataStr, -1)
+	idmatches := idPat.FindAllStringSubmatch(dataStr, -1)
+
+	for _, match := range matches {
+		if len(match) != 2 {
+			continue
+		}
+		resMatch := strings.TrimSpace(match[1])
+
+		for _, idmatch := range idmatches {
+			if len(idmatch) != 2 {
+				continue
+			}
+			resIdMatch := strings.TrimSpace(idmatch[1])
+
+			s1 := detectors.Result{
+				DetectorType: detectorspb.DetectorType_Tokeet,
+				Raw:          []byte(resMatch),
+			}
+			if verify {
+				req, err := http.NewRequestWithContext(ctx, "GET", "https://capi.tokeet.com/v1/user?account="+resIdMatch, nil)
+				if err != nil {
+					continue
+				}
+				req.Header.Add("Accept", "application/json")
+				req.Header.Add("Authorization", fmt.Sprintf("%s", resMatch))
```

Comment:
nit: Prefer to remove fmt.Sprintf here: `fmt.Sprintf("%s", resMatch`) and we can just use `..("Authorization", resMatch)...`